webrtc/src/rtp_transceiver/rtp_sender/mod.rs

```diff
@@ -275,8 +335,13 @@ impl RTCRtpSender {
 
     /// track returns the RTCRtpTransceiver track, or nil
     pub async fn track(&self) -> Option<Arc<dyn TrackLocal + Send + Sync>> {
-        let track = self.track.lock().await;
-        track.clone()
+        let encodings = self.track_encodings.lock().await;
```

Comment:
Yup, this is another pion-ism, and I dare say if we went down the ADT route for expressing whether it was simulcast or otherwise then perhaps this API could change too